doc/source/changes/version_0_22.rst.inc

```diff
@@ -105,11 +105,28 @@
 
     (closes :issue:`18`).
 
-* added another feature (see the :ref:`miscellaneous section <misc>` for details).
+* added load_example_data function to load datasets used in tutorial and be able to reproduce examples.
+  The name of the dataset must be provided as argument (see list of available datasets in
```

Comment:
I would write this instead: 
  
    The name of the dataset must be provided as argument (there is currently only one available dataset)